graphql-kotlin-federation/src/main/kotlin/com/expediagroup/graphql/federation/FederatedSchemaGeneratorHooks.kt

```diff
@@ -43,6 +43,7 @@ import kotlin.reflect.full.findAnnotation
  */
 open class FederatedSchemaGeneratorHooks(private val federatedTypeRegistry: FederatedTypeRegistry) : SchemaGeneratorHooks {
     private val directiveRegex = "(^#.+$[\\r\\n])?^directive @\\w+.+$[\\r\\n]*".toRegex(setOf(RegexOption.MULTILINE, RegexOption.IGNORE_CASE))
+    private val declaredDirectiveRegex = "@(?!extends)(?!external)(?!key)(?!provides)(?!requires)\\w+(?:\\(.*(?=\\))\\))?".toRegex(setOf(RegexOption.MULTILINE, RegexOption.IGNORE_CASE))
```

Comment:
lets rename those regexes to

* `directiveRegex` -> `directiveDefinitionRegex`
* `declaredDirectiveRegex` -> `customDirectivesRegex`
* `scalarRegex` -> `scalarDefinitionRegex`
* `emptyQuery` -> `emptyQueryRegex`

Reply:
alright wait